bitnami/discourse/README.md

```diff
@@ -438,6 +438,12 @@ Find more information about how to deal with common errors related to Bitnami’
 
 ## Upgrading
 
+### To 4.0.0
+
+The [Bitnami Discourse](https://github.com/bitnami/bitnami-docker-discourse) image was refactored and now the source code is published in GitHub in the [`rootfs`](https://github.com/bitnami/bitnami-docker-discourse/tree/master/2/debian-10/rootfs) folder of the container image repository.
+
+Full compatibility is not guaranteed due to the amount of involved changes, however no breaking changes are expected.
```

Comment:
I guess `skipInstall` is removed in this version. Maybe you want to add a comment on this change.

Reply:
Not removed, see above.